spec/pragma.dd

```diff
@@ -76,33 +76,65 @@ $(UL
 
 $(H3 $(LNAME2 crtctor, $(D pragma crt_constructor)))
 
-    $(P This pragma must directly precede an $(D extern(C)) function declaration
-        that must take no argument, even default ones.
-        The function this pragma applies to will be inserted in `.init_array`
-        or `.ctors`, depending on the target and compiler implementation.
-        It is equivalent to GCC's $(LINK2 https://gcc.gnu.org/onlinedocs/gcc-4.7.0/gcc/Function-Attributes.html, `__attribute__((constructor))`).
+    $(P Annotates a function so it is run after the C runtime library is initialized
+        and before the D runtime library is initialized.
     )
------------------
-__gshared int initCount;
 
-pragma(crt_constructor)
-extern(C) void initializer() { initCount += 1; }
------------------
+    $(P The function must:)
+
+    $(OL
+        $(LI be `extern (C)`)
+        $(LI not have any parameters)
```

Comment:
Missing mention of default parameters here. While it's obvious for a compiler developer why it wouldn't work, it might not be that obvious to the less informed reader.

Reply:
I thought about that, and specifically wrote "parameters" instead of "arguments". Default parameter declarations are placed in the declaration, and are patently obviously parameters.

The spec isn't meant as a tutorial for less informed readers. It's supposed to be precise, and come with a minimum of redundancy and extraneous verbiage.